inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/fake_quantize.hpp

```diff
@@ -61,4 +62,42 @@ class FakeQuantizeLayerTest : public testing::WithParamInterface<fqLayerTestPara
     int32_t  seed = 1;
 };
 
+
+//TODO after update all plugins remove *Revise types
+typedef std::tuple<
+        size_t,                         // fake quantize levels
+        std::vector<size_t>,            // fake quantize inputs shape
+        std::vector<float>,             // fake quantize (inputLow, inputHigh, outputLow, outputHigh) or empty for random
+        std::vector<float>,             // input generator data (low, high, resolution) or empty for default
+        ngraph::op::AutoBroadcastSpec   // fake quantize broadcast mode
+> fqSpecificParamsRevise;
+typedef std::tuple<
+        fqSpecificParamsRevise,
+        InferenceEngine::Precision,        // Net precision
+        InferenceEngine::Precision,        // Input precision
+        InferenceEngine::Precision,        // Output precision
+        InferenceEngine::Layout,           // Input layout
+        InferenceEngine::Layout,           // Output layout
+        InferenceEngine::SizeVector,       // Input shapes
+        LayerTestsUtils::TargetDevice,     // Device name
+
+        std::pair<std::string, std::map<std::string, std::string>> // Additional backend configuration and alis name to it
+> fqLayerTestParamsSetRevise;
+
+class FakeQuantizeLayerTestRevise : public testing::WithParamInterface<fqLayerTestParamsSetRevise>,
```

Comment:
Do we have a plan for plugins refactoring?

Reply:
Are you asking for Multiphasedapproach, then yes. Same like e.g.: Reshape which start from *57975* ticket.